kai/kai_logging.py

```diff
@@ -11,15 +11,16 @@
 )
 
 
-def process_log_dir_replacements(log_dir: str):
+def process_log_dir_replacements(log_dir: str) -> str:
     ##
     # We want to replace $pwd with the location of the Kai project directory,
     # this is needed to help with specifying from configuration
     ##
     if log_dir.startswith("$pwd"):
-        log_dir = log_dir.replace(
-            "$pwd", os.path.join(os.path.dirname(os.path.realpath(__file__)), "../")
-        )
+        current_directory = os.getcwd()
+        kai_project_directory = os.path.abspath(os.path.join(current_directory, ".."))
```

Comment:
We need to change this and get rid of the "..", that was needed in past when we were basing the directory to the location of "__file__".